src/frontends/common/include/openvino/frontend/node_context.hpp

```diff
@@ -48,6 +48,11 @@ class FRONTEND_API NodeContext {
         FRONT_END_NOT_IMPLEMENTED(get_input);
     }
 
+    /// \brief Returns const input with a given index
+    virtual Any const_input(int idx) const {
+        FRONT_END_NOT_IMPLEMENTED(const_input);
+    }
+
```

Comment:
not sure that we need this method in the common place. Is this really needed?

Reply:
Code in frontend/common that handles OpExtension refers to base NodeContext class https://github.com/openvinotoolkit/openvino/pull/16434/files#diff-a441f7a33d0e530dc1b19988cda0dd386a5cde256bf34871dc30217b9de8ba10R243. That's why I thought it would be good to have it here.